### ISSUE TITLE
False positive EANSIBLE0013 when using globs

### ISSUE THREAD

**Issue by allanlewis** [willthames/ansible-lint#163]
# Issue Type
- Bug report

# Ansible and Ansible Lint details
```
ansible --version
2.1.0.0
ansible-lint --version
3.0.1
```
- ansible installation method: pip
- ansible-lint installation method: pip

# Desired Behaviour
`EANSIBLE0013` should not be raised for `shell` tasks that use globs.

# Actual Behaviour
`EANSIBLE0013` is raised for this task:
```yaml
    - name: Deploy ssh key for root
      shell: rsync /home/foo/.ssh/id_rsa* /root/.ssh/
      args:
        creates: /root/.ssh/id_rsa
```
Output of `ansible-lint`:
```
[ANSIBLE0013] Use shell only when shell functionality is required
<path>.yml:<n>
Task/Handler: Deploy ssh key for root
```

**Comment by allanlewis** [comment]
I've also seen this with shell commands using brace expansion, e.g. `foo{bar,baz}`.

**Comment by willthames** [comment]
I might extend it to cope with `*` but this is just ansible-lint correctly complaining but at the wrong thing ;)

In this case ansible-lint should be moaning at you for using shell with rsync (rather than the synchronize module), and in a perfect world it would suggest you use `copy` and `with_fileglob` 

Because ansible-lint works at the pretemplate level, coping with braces might be trickier. 

Tagging such things `skip_ansible_lint` is the quick workaround though.

**Comment by ssbarnea** [comment]
I do not find the `skip_ansible_lint` approach as a sane one because it does disable linting completely. We need to be able to disable individual linting errors instead of disabling linting for an entire task.

**Comment by willthames** [comment]
PRs accepted for `# noqa` but not sure how easy it would be to implement with the various different types of checks (easy to make noqa work for line level checks, harder on task level checks where the comment is parsed out before the yaml is even inspected)
